src/stories/Customization.stories.mdx

```diff
@@ -48,9 +48,79 @@ as you see fit. For instance, if you'd like to change the color of the curve of
+| `--h5w-line--auxColor`   | `cornflowerblue` | Color of auxiliary curves               |
+
+#### Heatmap
+
+| Name                                    | Default                                      | Description                               |
+| --------------------------------------- | -------------------------------------------- | ----------------------------------------- |
+| `--h5w-colorBar-ticks--color`           | `var(--h5w-ticks--color, gray)`              | Color of color bar ticks                  |
+| `--h5w-colorBar-tickLabels--color`      | `var(--h5w-tickLabels--color, #333)`         | Text color of color bar tick labels       |
+| `--h5w-colorBar-tickLabels--fontFamily` | `var(--h5w-tickLabels--fontFamily, inherit)` | Font family of color bar tick labels      |
+| `--h5w-colorBar-tickLabels--fontSize`   | `0.75em`                                     | Font size of color bar tick labels        |
+| `--h5w-colorBar-bounds--color`          | `#333`                                       | Font size of color bar min and max bounds |
+| `--h5w-colorBar-bounds--fontSize`       | `0.875em`                                    | Font size of color bar min and max bounds |
+
+#### Matrix
+
+| Name                                | Default                                    | Description                            |
+| ----------------------------------- | ------------------------------------------ | -------------------------------------- |
+| `--h5w-matrix--color`               | `inherit`                                  | Text color of matrix vis               |
+| `--h5w-matrix--fontFamily`          | `monospace`                                | Font family of matrix vis              |
+| `--h5w-matrix--fontSize`            | `inherit`                                  | Font size of matrix vis                |
+| `--h5w-matrix--cellBg`              | `transparent`                              | Background colour of cells (1/2)       |
+| `--h5w-matrix--cellBgAlt`           | `#eee`                                     | Background colour of cells (2/2)       |
+| `--h5w-matrix--indexCellBg`         | `#eee`                                     | Background colour of index cells (1/2) |
+| `--h5w-matrix--indexCellBgAlt`      | `lightgray`                                | Background colour of index cells (2/2) |
+| `--h5w-matrix--indexCellColor`      | `var(--h5w-matrix--color, inherit)`        | Text color of index cells              |
+| `--h5w-matrix--indexCellFontFamily` | `var(--h5w-matrix--fontFamily, monospace)` | Font family of index cells             |
+| `--h5w-matrix--indexCellFontSize`   | `var(--h5w-matrix--fontSize, inherit)`     | Font size of index cells               |
+| `--h5w-matrix--indexCellFontWeight` | `600`                                      | Font weight of index cells             |
+| `--h5w-matrix--anchorCellBg`        | `silver`                                   | Background colour of anchor cell       |
```

Comment:
I wasn't sure if I should go with `--h5w-matrix-indexCell--fontFamily` like for the Heatmap's color bar. What do you think?

Reply:
Yeah `--h5w-matrix-indexCell--fontFamily` pleases me more.

You could then have `--h5w-tooltip-guide--color` as well.